apps/web/src/studio/components/workflows/step-editor/WorkflowStepEditorControlsPanel.tsx

```diff
@@ -19,6 +19,7 @@ export type OnChangeType = 'step' | 'payload';
 interface IWorkflowStepEditorControlsPanelProps {
   step: any;
   workflow: any;
+  error?: any;
```

Comment:
Please import the `ValidationError` type from `@novu/framework` to use here.

Also, I think this be a plural `errors` that this component receives.